contracts/contracts/token/OUSD.sol

```diff
@@ -21,51 +15,71 @@ import { Governable } from "../governance/Governable.sol";
  * rebasing design. Any integrations with OUSD should be aware.
  */
 
-contract OUSD is Initializable, InitializableERC20Detailed, Governable {
-    using SafeMath for uint256;
-    using StableMath for uint256;
-
+contract OUSD is Governable {
     event TotalSupplyUpdatedHighres(
         uint256 totalSupply,
         uint256 rebasingCredits,
         uint256 rebasingCreditsPerToken
     );
     event AccountRebasingEnabled(address account);
     event AccountRebasingDisabled(address account);
+    event Transfer(address indexed from, address indexed to, uint256 value);
+    event Approval(
+        address indexed owner,
+        address indexed spender,
+        uint256 value
+    );
 
     enum RebaseOptions {
         NotSet,
-        OptOut,
-        OptIn
+        StdNonRebasing,
+        StdRebasing,
+        YieldDelegationSource,
+        YieldDelegationTarget
     }
 
+    // Add slots to align with deployed OUSD contract
+    uint256[154] private _gap;
     uint256 private constant MAX_SUPPLY = ~uint128(0); // (2^128) - 1
     uint256 public _totalSupply;
     mapping(address => mapping(address => uint256)) private _allowances;
     address public vaultAddress = address(0);
     mapping(address => uint256) private _creditBalances;
-    uint256 private _rebasingCredits;
+    uint256 private _rebasingCredits; // Sum of all rebasing credits (_creditBalances for rebasing accounts)
     uint256 private _rebasingCreditsPerToken;
-    // Frozen address/credits are non rebasing (value is held in contracts which
-    // do not receive yield unless they explicitly opt in)
-    uint256 public nonRebasingSupply;
-    mapping(address => uint256) public nonRebasingCreditsPerToken;
+    uint256 public nonRebasingSupply; // All nonrebasing balances
+    mapping(address => uint256) private alternativeCreditsPerToken;
     mapping(address => RebaseOptions) public rebaseState;
     mapping(address => uint256) public isUpgraded;
+    mapping(address => address) public yieldTo;
+    mapping(address => address) public yieldFrom;
 
     uint256 private constant RESOLUTION_INCREASE = 1e9;
 
     function initialize(
-        string calldata _nameArg,
-        string calldata _symbolArg,
+        string calldata,
+        string calldata,
         address _vaultAddress,
         uint256 _initialCreditsPerToken
-    ) external onlyGovernor initializer {
-        InitializableERC20Detailed._initialize(_nameArg, _symbolArg, 18);
+    ) external onlyGovernor {
```

Comment:
nit: Should we instead try to reset the `initialized` var to zero rather than dropping it there?

Reply:
nvm, Just saw that we are getting rid of `Initializable`